### PR TITLE
Create 2024-07-15-get-together-ome-transforms

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

**Comment by thewtex** [comment]
@MSanKeys963 @joshmoore @bogovicj please take a look 🙏

**Review by joshmoore — APPROVED** [review]
:heart: Thanks, @thewtex!

File: _posts/2024-07-15-get-together-ome-transforms.markdown [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## Event Details
+
+- **Dates:** Friday, July 26th - Sunday, July 28th, 2024
+- **Location:** Hybrid event at the University of North Carolina-Chapel Hill, via Google Meet videoconferencing, Image.sc Island Gather.Town virtual space, and Image.sc Zulip Chat.
+- **Cost:** [Registration] is free!
+
+**[Register now and add the event to your calendar!](https://insightsoftwareconsortium.github.io/GetYourBrainTogether/HCK03_2024_UNC_Hybrid/)**
+
+[Registration]: https://forms.gle/LL4quQsbSWawKYSa6
```
Comment (d-v-b) [inline _posts/2024-07-15-get-together-ome-transforms.markdown:41]: is the duplication of the registration link intended?

Comment (joshmoore) [reply _posts/2024-07-15-get-together-ome-transforms.markdown:41]: I think it's intended in the sense that it let's you use "`[Register]`" and "`[Registration]`" in the text without the longer-form of "`[Register][Registration]`".

**Pull request merged**